multichain-api-gateway/src/lib.rs

```diff
@@ -2,12 +2,13 @@ use std::{collections::HashMap, net::TcpListener, str};
 
 use actix_web::{
     dev::Server,
+    http::{header::HeaderMap, Method},
     web,
-    web::{Data, Json},
+    web::{Bytes, Data, Json},
     App, HttpRequest, HttpServer, Responder,
 };
+use awc::Client;
```

Comment:
I decided to try replace "reqwest" client with actix-web client (awc) since we use actix-web server. It takes almost the same lines of code.
Let me know what you think; it's easy to rollback to "reqwest" if needed.